apps/sms/views/shared/js/startup.js

```diff
@@ -113,21 +119,18 @@ var Startup = exports.Startup = {
 
       // Init UI Managers
       TimeHeaders.init();
-      ConversationView.init();
-      MessagingClient.init(App.instanceId);
-      MozMobileConnectionsClient.init(App.instanceId);
       Information.initDefaultViews();
 
+      return ConversationView.init();
```

Comment:
I think this change make sense, but I can't figure out a better way to verify this in unit test(we can stub the Conversation since it's lazy loaded). Any good idea for this?